pkg/resources/stream.go

```diff
@@ -228,9 +228,9 @@ func ReadStream(d *schema.ResourceData, meta interface{}) error {
 }
 
 // DeleteStream implements schema.DeleteFunc
-func DeleteStream(d *schema.ResourceData, meta interface{}) error {
+func DeleteStream(data *schema.ResourceData, meta interface{}) error {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R014: *schema.ResourceData parameter of CreateFunc, ReadFunc, UpdateFunc, or DeleteFunc should be named d